src/io/flutter/editor/FlutterCompletionContributor.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.editor;
+
+import com.intellij.codeInsight.lookup.LookupElementBuilder;
+import com.intellij.openapi.project.Project;
+import com.intellij.util.ui.ColorIcon;
+import com.intellij.util.ui.JBUI;
+import com.jetbrains.lang.dart.ide.completion.DartServerCompletionContributor;
+import org.apache.commons.lang.StringUtils;
+import org.dartlang.analysis.server.protocol.CompletionSuggestion;
+import org.dartlang.analysis.server.protocol.Element;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.*;
+import java.util.Objects;
+
+
+public class FlutterCompletionContributor extends com.jetbrains.lang.dart.ide.completion.DartCompletionExtension {
```

Comment:
add `com.jetbrains.lang.dart.ide.completion.DartCompletionExtension` to the imports?

Reply:
Whoops.  Left over from some early fiddles.  Fixed.